zubhub_frontend/zubhub/src/views/project_details/ProjectDetails.jsx

```diff
@@ -431,6 +455,11 @@ function ProjectDetails(props) {
             style: {
               backgroundColor: 'transparent',
               boxShadow: 'none',
+              width: '80%',
+              height: '40%',
+              display: 'flex',
+              justifyContent: 'center',
+              alignItems: 'center',
```

Comment:
Avoid inline styles. And also set max-width and max height or use clamp for width or height so it doesn't look odd in big screens